docs/guides/expectations/how_to_create_and_edit_expectations_with_a_profiler.md

```diff
@@ -9,10 +9,230 @@ This guide will help you create a new Expectation Suite by profiling your data w
 <Prerequisites>
 
 - Configured a [Data Context](../../tutorials/getting_started/initialize_a_data_context.md).
+- Configured a [Datasource](../../tutorials/getting_started/connect_to_data.md)
 
 </Prerequisites>
 
-:::caution
-The UserConfigurableProfiler is not currently configured to work with the 0.13 Experimental API.
-We expect to update this in the near future.
+:::note
+The User Configurable Profiler makes it easier to produce a new Expectation Suite by building out a bunch of Expectations for your data.
+
+These Expectations are deliberately over-fitted on your data e.g. if your table has 10,000 rows, the profiler will produce an Expectation with the following config:
+```json
+{
+      "expectation_type": "expect_table_row_count_to_be_between",
+      "kwargs": {
+        "min_value": 10000,
+        "max_value": 10000
+      },
+      "meta": {}
+    }
+```
+
+Thus, the intention is for this Expectation Suite to be edited and updated to better suit your specific use case - it is not specifically intended to be used as is.
+:::
+
+:::note
+You can access this same functionality from the Great Expectations CLI by running
+```console
+great_expectations --v3-api suite new --profile
+```
+
+If you go that route, you can follow along in the resulting Jupyter Notebook instead of using this guide.
 :::
+
+
+## Steps
+
+### 1. Load or create your Data Context
+
+Load an on-disk Data Context via:
+```python
+    from great_expectations.data_context.data_context import DataContext
+
+    context = DataContext(
+        context_root_dir='path/to/my/context/root/directory/great_expectations'
+    )
+```
```

Comment:
Could we left-align this block? Pedantic for sure but this will make all code blocks consistent 😄